Docs/user_guide/settings.md

```diff
@@ -142,3 +142,22 @@ The following environment variables can be set before the application is started
 - `SLICER_EXTENSIONS_MANAGER_SERVER_URL`: URL of the extensions manager backend with the `/api` path. Default value is retrieved from the settings using the key `Extensions/ServerUrl`.
 - `SLICER_EXTENSIONS_MANAGER_FRONTEND_SERVER_URL`: URL of the extension manager frontend displaying the web page. Default value is retrieved from the settings using the key `Extensions/FrontendServerUrl`.
 - `SLICER_EXTENSIONS_MANAGER_SERVER_API`: Supported values are `Midas_v1` and `Girder_v1`. Default value is hard-coded to `Girder_v1`.
+
+### Qt built-in command-line options
+
+Slicer application supports specific command-line options that allow modifying the way Qt will interact with the windowing system.
+
+Examples of options:
+- `-widgetcount`, prints debug message at the end about number of widgets left undestroyed and maximum number of widgets existed at the same time.
```

Comment:
Are there any better examples of options that could be actually useful? For example, controlling DPI awareness could be useful. Or setting the window geometry or display number.